GiphySearch/ViewController.swift

```diff
@@ -9,19 +9,19 @@ import UIKit
 
 class ViewController: UIViewController {
     
-    @IBOutlet weak var collectionView: UICollectionView!
-    @IBOutlet weak var searchBar: UISearchBar!
+    @IBOutlet private var collectionView: UICollectionView!
+    @IBOutlet private var searchBar: UISearchBar!
     
-    var network = GifNetwork()
-    var gifs = [Gif]()
-    var currentPage: Int?
+    private var network = GifNetwork()
```

Comment:
Privacy matters not only in real life, do not expose any information that doesn't need to be exposed